recipe/build.sh

```diff
@@ -38,6 +38,8 @@ cmake \
   -DVTK_USE_SYSTEM_CONSTANTLY=ON \
   -DVTK_USE_SYSTEM_AUTOBAHN=ON \
   -DVTK_OPENGL_HAS_OSMESA:BOOL=ON \
+  -DOSMESA_LIBRARY=${PREFIX}/lib/libOSMesa32.so \
+  -DOSMESA_INCLUDE_DIR=${PREFIX}/include \
```

Comment:
- OSMESA_INCLUDE_DIR should already be found as CMAKE_PREFIX_PATH is set
- OSMESA_LIBRARY is not found as FindOSMesa looks for libOSMesa
this could be proposed as a modification of their FindOSMesa.cmake:
https://gitlab.kitware.com/vtk/vtk/blob/master/CMake/FindOSMesa.cmake
```find_library(OSMESA_LIBRARY OSMesa OSMesa32 PATHS```

Reply:
> Why libosmesa32 ? isnt it a 64 bit distro ?

It's not about the distro. From the docs

> This option allows the size of the color channel in bits to be specified. By default, an 8-bit channel will be used, and the driver will be named libOSMesa. Other options are 16- and 32-bit color channels, which will add the bit size to the library name. For example, --with-osmesa-bits=16 will create the libOSMesa16 library with a 16-bit color channel.

Reply:
ok, https://gitlab.kitware.com/vtk/vtk/merge_requests/3289